android/java/org/chromium/chrome/browser/crypto_wallet/activities/AddAccountActivity.java

```diff
@@ -241,12 +172,85 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                         }
                         mPrivateKeyControl.setText(content);
                     } catch (Exception ex) {
-                        Log.e("NTP", ex.getMessage());
+                        Log.e(TAG, "NTP", ex);
```

Comment:
I don't know what `NTP` stands for.

Reply:
Seems like old logs and @SergeyZhukovsky may want to remove this as well. NTP is New Tab Page I guess.

Reply:
yeah, probably just a copy/paste from New Tab Page logic somewhere. I would keep it and replace to the correct function name.

Reply:
No problem, I'm adding a more meaningful message.